### PR TITLE
feat(replays): Document the new search properties available for the Replay DOM Search feature

### LINKED ISSUE #6464: Update [searchable properties docs](https://docs.sentry.io/product/sentry-basics/search/searchable-properties/)
Need to update the docs site to account for the new searchable properties that DOM Search uses.

The current thinking is to use the "explicit key→value" style. ex: `replay.click.tag:button replay.click.class:btn-primary`

See also: https://github.com/getsentry/sentry/issues/45741

### PR DESCRIPTION
**Pull request by ryan953**
Related to https://github.com/getsentry/sentry/pull/45744
Related to https://github.com/getsentry/sentry/pull/45763

Related to https://github.com/getsentry/sentry/pull/47184
Related to https://github.com/getsentry/sentry/pull/47162

Fixes #6464

### CODE REVIEW

**Comment by lizokm** [comment]
Hi hi! @ryan953 Would you like me to take a look at this while it's still in draft or should I hold off for now?

**Review by cmanallen — CHANGES_REQUESTED** [review]

File: src/docs/product/sentry-basics/search/searchable-properties.mdx [hunk 1]
```diff
@@ -70,6 +70,19 @@ The Xs in these columns of the table indicate whether the property is an event p
...
+    | replay_click.alt    | string        | The alt attribute of the HTML element.                         |
+    | replay_click.class  | array[string] | An array of HTML element classes.                              |
+    | replay_click.id     | string        | The ID of an HTML element.                                     |
+    | replay_click.label  | string        | The aria-label attribute of an HTML element.                   |
+    | replay_click.role   | string        | The role of an HTML element.                                   |
+    | replay_click.tag    | string        | Valid HTML5 tag name.                                          |
+    | replay_click.testid | string        | The data-testid of an HTML element. (omitted from public docs) |
+    | replay_click.text   | string        | The text-content of an HTML element.                           |
+    | replay_click.title  | string        | The title attribute of an HTML element.                        |
+
```
[the remark this answers is not in the document]
Comment (ryan953) [reply src/docs/product/sentry-basics/search/searchable-properties.mdx:85]: is `selector` in-scope now?! woo!

[the remark this answers is not in the document]
Comment (cmanallen) [reply src/docs/product/sentry-basics/search/searchable-properties.mdx:85]: @ryan953 Sure is :D

**Comment by ryan953** [comment]
@lizokm I think this is ready for your eyes. We've settled on the list of fields, no new fields will be added so we can describe what we have.

One thing that might change is the the prefix `replay_click` might become just `click`... i'm not sure who's going to come down and decide that change or not. so TBD for now.

**Review by bruno-garcia — APPROVED** [review]

File: src/docs/product/sentry-basics/search/searchable-properties.mdx [hunk 2]
```diff
@@ -191,6 +192,15 @@ Release properties overlap with event and issue search properties, so they don't
...
+| click.role | The `role` of an element that was clicked. For example `button` would match both `<button>Save</button>` and `<a role="button">Submit</a>` | | | x | string |
+| click.selector | An element identified using a subset of CSS selector syntax. For example `span#section-1` or `span.active` or `span[role=button]` or `span#section-1.active[role=button]` would all match the element `<span id="section-1" class="active" role="button"/>`. CSS combinators, pseudo selectors, and attr selectors other than `=` are not supported. | | | x | string |
+| click.tag | The tag name of an element that was clicked. For example `input` would match `<input name="username" />` | | | x | string |
+| click.textContent | The immediate `textContent` of an element that was clicked. For example `Save` would match `<button>Save</button>` but would not match `<button><h1>Save</h1></button>` | | | x | string |
+| click.title | The `title` of an element that was clicked. For example `Save this comment` would match the element `<a title="Save this comment" class="btn btn-primary">Save</a>` | | | x | string |
```
Comment (lizokm) [inline src/docs/product/sentry-basics/search/searchable-properties.mdx:203]:
```suggestion
| click.alt | The `alt` of an element that was clicked. For example, `"a good dog"` would match the element `<img src="/lassie.jpeg" alt="a good dog" />` | | | x | string |
| click.class | The `class` of an element that was clicked. No leading `.` is necessary. For example, `btn-primary` would match the element `<a class="btn btn-primary">Save</a>` | | | x | string |
| click.id | The `id` of an element that was clicked. No leading `#` is necessary. For example, `reset-password` would match the element `<a id="reset-password">Reset</a>` | | | x | string |
| click.label | The `aria-label` of an element that was clicked. For example, `Expand` would match the element `<button aria-label="Expand"><img src="/icons/expand.png"/></button>` | | | x | string |
| click.role | The `role` of an element that was clicked. For example, `button` would match both `<button>Save</button>` and `<a role="button">Submit</a>` | | | x | string |
| click.selector | An element identified using a subset of CSS selector syntax. For example, `span#section-1` or `span.active` or `span[role=button]` or `span#section-1.active[role=button]` would all match the element `<span id="section-1" class="active" role="button"/>`. Note that, CSS combinators, pseudo selectors, and attr selectors other than `=` are not supported. | | | x | string |
| click.tag | The tag name of an element that was clicked. For example, `input` would match `<input name="username" />` | | | x | string |
| click.textContent | The immediate `textContent` of an element that was clicked. For example, `Save` would match `<button>Save</button>` but wouldn't match `<button><h1>Save</h1></button>` | | | x | string |
| click.title | The `title` of an element that was clicked. For example, `Save this comment` would match the element `<a title="Save this comment" class="btn btn-primary">Save</a>` | | | x | string |
```

**Review by lizokm — APPROVED** [review]
Just a few small tweaks.

**Pull request merged**